spec/features/platform_admin_can_view_all_vendors_spec.rb

```diff
@@ -0,0 +1,140 @@
+require "rails_helper"
+require "factory_helper"
+
+feature "Admin can view all events from Admin Dashboard" do
+  before do
+    admin = User.create(first_name: "Dave",
+    last_name: "Shim",
+    username: "admin",
+    email: "dave@daveshim.com",
+    password: "password",
+    role: "admin")
+
+    platform_role = Role.create(
+    name: "platform_admin"
+    )
+
+    admin.roles << platform_role
+
+    build_test_data
+
+    allow_any_instance_of(ApplicationController)
+    .to receive(:current_user).and_return(admin)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.